src/Features/Protocol/LanguageServices/Microsoft.CodeAnalysis.Protocol.LanguageServices.csproj

```diff
@@ -5,12 +5,27 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.Protocol.LanguageServices</RootNamespace>
```

Comment:
>Microsoft.CodeAnalysis.Protocol.LanguageServices [](start = 19, length = 48)

Namespace: `Microsoft.CodeAnalysis.LanguageServer`

Reply:
The reason it was protocol first was because it seemed to be a higher level concept than language server.  But I'm fine changing it to match the vs package.